API.md

```diff
@@ -3,14 +3,15 @@
 ### Table of Contents
 
 -   [DOM Interaction Helpers](#dom-interaction-helpers)
+    -   [blur](#blur)
```

Comment:
Can you reset the changes here to `API.md`? I'll regenerate it before publishing (this way the API.md on master roughly continues to match what actual users are using)...

Reply:
done